py_zipkin/zipkin.py

```diff
@@ -143,6 +154,17 @@ def __init__(
         if self.sample_rate is not None and not (0.0 <= self.sample_rate <= 100.0):
             raise ZipkinError('Sample rate must be between 0.0 and 100.0')
 
+        if not set(include).issubset(set(STANDARD_ANNOTATIONS.keys())):
```

Comment:
I considered this, but I thought it to be premature optimization.

```
$ python -m timeit -s 'x={"a":{"a1","a2","a3"},"b":{"b1","b2","b3"},"c":{"c1","c2","c3"},"d":{"d1","d2","d3"},"e":{"e1","e2","e3"}}' 'y=set(x.keys())'
1000000 loops, best of 3: 0.476 usec per loop
```

But since I don't think it hurts readability to make the change, i'll go ahead and make it.